grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java

```diff
@@ -100,21 +95,21 @@ public TasksController(Service spark, TaskManager manager) {
     })
     private Json getTasks(Request request, Response response) {
         TaskStatus status = null;
-        String className = request.queryParams(TASK_CLASS_NAME_PARAMETER);
-        String creator = request.queryParams(TASK_CREATOR_PARAMETER);
+        String className = request.queryParams(REST.Request.TASK_CLASS_NAME_PARAMETER);
```

Comment:
YESSSSS!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! I hate static imports of strings and enums.